apps/prairielearn/src/server.js

```diff
@@ -2161,17 +2161,21 @@ export async function startServer() {
   });
   server.on('connection', () => connectionCounter.add(1));
 
-  opentelemetry.createObservableValueGauges(
-    meter,
-    'http.connections.active',
-    {
-      valueType: opentelemetry.ValueType.INT,
-      interval: 1000,
-    },
-    () => {
-      return util.promisify(server.getConnections.bind(server))();
-    },
-  );
+  // Hack to get us running in Bun, which doesn't currently support `getConnections`:
+  // https://github.com/oven-sh/bun/issues/4459
+  if (server.getConnections) {
```

Comment:
So this is going to make is to we can run in bun only in local dev, right? maybe that's something you should note in the comment.

Reply:
No, that's not true. One is free to run it in production with Bun, they just wouldn't get this specific metric. We certainly won't be doing that, of course.